src/Sulu/Bundle/WebsiteBundle/DependencyInjection/SuluWebsiteExtension.php

```diff
@@ -29,9 +30,11 @@ class SuluWebsiteExtension extends Extension implements PrependExtensionInterfac
 {
     public function prepend(ContainerBuilder $container)
     {
-        $container->prependExtensionConfig('twig', [
-            'exception_controller' => 'sulu_website.exception.controller:showAction',
-        ]);
+        if ($container->hasExtension('twig') && class_exists(ExceptionController::class)) {
+            $container->prependExtensionConfig('twig', [
+                'exception_controller' => null,
```

Comment:
Does the new way also work with Symfony 4.4? Then this setting would not be necessary at all, would it? What's the point of just setting it to `null`?

Reply:
@danrot the new way does work for 4.4 but that it works we need to set this to null:

https://github.com/symfony/symfony/blob/cd66cd57a05fac7deaacf45c3b6ae5282e04d344/src/Symfony/Bundle/TwigBundle/DependencyInjection/Configuration.php#L38-L42

Reply:
Okay, that's weird :see_no_evil: